src/intellisense/texDocFormatter.ts

```diff
@@ -0,0 +1,64 @@
+
+import * as vscode from 'vscode';
+import * as Prettier from "prettier";
+import { prettierPluginLatex } from "@unified-latex/unified-latex-prettier";
+
+
+/*
+    * @param {string} source - The LaTeX source code to be formatted.
+    * @param {object} options - The options for Prettier.
+    * @returns {Promise<string>} - The formatted LaTeX source code.
+*/
+// https://github.com/siefkenj/latex-parser-playground/blob/master/src/async-worker/parsing-worker.ts#L35-L43
+async function format(source = "", options = {}) {
+    return Prettier.format(source, {
+        printWidth: 80,
```

Comment:
what is this magic number "80" for? why?

Reply:
This number is the length of formated string length, like this 
![image](https://github.com/iamhyc/Overleaf-Workshop/assets/51881852/fe945760-bbd0-46d5-af7f-ccf905805ed6)


And it is 80 in default, it is revealed for future application (if any)

Reply:
The unit is space.